.travis.yml

```diff
@@ -12,9 +12,7 @@ matrix:
           cache: {}
         - python: '3.4'
         - python: '3.5'
-        - python: 'nightly'
```

Comment:
I could, but I prefer to delete the code and make git remember what was there before.